scout/server/blueprints/variant/controllers.py

```diff
@@ -352,8 +353,10 @@ def observations(store, loqusdb, case_obj, variant_obj):
         if user_institutes_ids.isdisjoint(other_institutes):
             # If the user does not have access to the information we skip it
             continue
-        document_id = parse_document_id(chrom, str(pos), ref, alt, var_type, case_id)
-        other_variant = store.variant(document_id=document_id)
+
+        other_variant = store.variant(
```

Comment:
Agreed, that is convenient, but still not going to work for all returned SVs, right? loqusdb is destructive in its encoding, so similar, but not identical SVs from different cases will be mapped to the same loqusdb entry, and can not be easily recovered. An approach can be to return links to the exact SV variant iff it exists with the same id in the other case(s), but return the case name and link to the case page if the variant could not be easily identified, like just a jinja template fix. Let me know if you want me to try to draft something for you to check instead!

Reply:
I had the feeling the matter wasn't so easy to fix (therefore the title of the PR!). Checked now the [code on loqus](https://github.com/moonso/loqusdb/blob/a6b7ee426357dd493eb31777ded4d6246753c484/loqusdb/commands/view.py#L120) and as you say, all SVs matching those coordinates are returned, regardless of the SV type. I'll try to fix accordingly!